res2df/equil.py

```diff
@@ -342,28 +342,28 @@ def equil_main(args) -> None:
 
 
 def equil_reverse_main(args) -> None:
-    """Entry-point for module, for command line utility for CSV to Eclipse"""
-    logger = getLogger_ecl2csv(  # pylint: disable=redefined-outer-name
+    """Entry-point for module, for command line utility for CSV to resdata"""
```

Comment:
I guess this writes include file output, so we should say `include file` instead of `resdata`.